.gitignore

```diff
@@ -1,2 +1,3 @@
 .venv
 __pycache__
+.vscode
```

Comment:
Ezt miért adtad az ignore listához? Szerintem @molnarbence szándékosan tette verziókezeltté. Másrészt, ahhoz, hogy tényleg ignorálódjon, szerintem ki is kellene törölni `git rm --cached` paranccsal (gondolom, lokálban megőriznéd). De szerintem inkább ne legyen ignorálva.

Reply:
Jó régen adtam hozzá megszokásból, amikor még nem voltak benne a közös formázás stb. settingek, de kiveszem majd akkor, köszi, hogy szóltál.